uqcsbot/utils/uq_course_utils.py

```diff
@@ -332,52 +327,31 @@ def get_current_exam_period():
     return start_datetime, end_datetime
 
 
-def get_course_assessment_page(
-    course_names: List[str], offering: Optional[Offering]
-) -> str:
-    """
-    Determines the course ids from the course names and returns the
-    url to the assessment table for the provided courses
-    """
-    profile_ids = map(
-        lambda course: str(get_course_profile_id(course, offering=offering)),
-        course_names,
-    )
-    return BASE_ASSESSMENT_URL + ",".join(profile_ids)
-
-
 def get_course_assessment(
```

Comment:
Returns a list of assessment items but the name of the function and the docstring somewhat suggests that a single assessment is returned. Changing either the name or changing the docstring would be helpful but isn't required.